cli/yamls.py

```diff
@@ -111,7 +111,7 @@ def _dict_flattener(ord_dict):
     def dict_builder(_dic, parent_key=''):
         for key, value in _dic.iteritems():
             if parent_key:
-                key = parent_key + '.' + key
+                key = parent_key + '.' + str(key)
```

Comment:
I would like to see a test with an integer key. So in the future if someone change this function, ensure that still expecting the same result.